os/include/tinyara/lcd/st7785.h

```diff
@@ -45,13 +45,17 @@
 #if defined(CONFIG_LCD_PORTRAIT) || defined(CONFIG_LCD_RPORTRAIT)
 #define LCD_XRES CONFIG_LCD_XRES
 #define LCD_YRES CONFIG_LCD_YRES
-#define LCD_CASET 0x00EF
-#define LCD_RASET 0x013F
+#define LCD_CASET1 0x00
```

Comment:
Could you add more comments to the setting?
Such as below
/* MV = 0, Column = 0x00EF = (240 - 1), Row = 0x013F = (320 - 1) */

Reply:
It will help user to understand those meaning easier